### PR TITLE
Add deployment Dockerfile

### PR DESCRIPTION
**Pull request by mihails-strasuns**

### CODE REVIEW

**Comment by mihails-strasuns** [comment]
Green

**Comment by nemanja-boric-sociomantic** [comment]
Thank you!

File: .travis.yml [hunk 1]
```diff
@@ -51,6 +54,12 @@ jobs:
         - <<: *test-matrix
           env: DMD=2.078.* F=devel COV=1
 
+        # Test deployment docker image generation
+        - stage: Test
+          script:
+              - docker build --build-arg DMD=2.071.2.s* --build-arg DIST=xenial
+                -t dlsnode -f docker/Dockerfile.dlsnode .
```
Comment (nemanja-boric-sociomantic) [inline .travis.yml]: Can we bump this to 2.078 as well?

Comment (mihails-strasuns) [reply .travis.yml]: Sure. updated

**Pull request merged**